installation.md

```diff
@@ -96,6 +101,22 @@ Once privileges are granted, the `kubevirt.yaml` can be deployed:
 kubectl apply -f https://github.com/kubevirt/kubevirt/releases/download/${RELEASE}/kubevirt.yaml
 ```
 
+### From Service Catalog as an APB
+
+You can find KubeVirt in OpenShift Service Catalog and install it from there.
+In order to do that please follow documentation of
+[KubeVirt APB repository](https://github.com/ansibleplaybookbundle/kubevirt-apb).
+
+### Using Ansible playbooks
+
+There is project [kubevirt-ansible](https://github.com/kubevirt/kubevirt-ansible)
+which provides a collection of playbooks to install KubeVirt on OpenShift or
+Kubernetes cluster.
+
+Supported scenarions
```

Comment:
I am considering to remove this section, because scenarios are described in kubevirt-ansible repository.